src/views/esp/view_esp.cpp

```diff
@@ -85,6 +85,85 @@ namespace big
 			if (distance < g.esp.box_render_distance[1] && distance > g.esp.box_render_distance[0] && g.esp.box)
 				draw_list->AddRect({esp_x - (62.5f * multplr), esp_y - (175.f * multplr)}, {esp_x - (62.5f * multplr) + (125.f * multplr), esp_y - (175.f * multplr) + (350.f * multplr)}, esp_color);
 
+			if (distance < g.esp.skeleton_render_distance[1] && distance > g.esp.skeleton_render_distance[0] && g.esp.skeleton)
+			{
+				std::vector<PedBones> bone_ids = // size: 18
+				{
+					PedBones::SKEL_Head, 
+					PedBones::SKEL_Neck_1,
+					PedBones::SKEL_Spine_Root, 
+					PedBones::SKEL_Pelvis, 
+					PedBones::SKEL_L_Thigh, 
+					PedBones::SKEL_R_Thigh, 
+					PedBones::SKEL_L_Calf, 
+					PedBones::SKEL_R_Calf, 
+					PedBones::SKEL_L_Foot, 
+					PedBones::SKEL_R_Foot, 
+					PedBones::SKEL_L_Clavicle, 
+					PedBones::SKEL_R_Clavicle, 
+					PedBones::SKEL_L_UpperArm, 
+					PedBones::SKEL_R_UpperArm, 
+					PedBones::SKEL_L_Forearm, 
+					PedBones::SKEL_R_Forearm, 
+					PedBones::SKEL_L_Hand, 
+					PedBones::SKEL_R_Hand
+				};
+
+				std::vector<rage::fvector4> bones = {};
+				const size_t skel_size            = 18;
+
+				for (int i = 0; i < skel_size; i++)
+				{
+					rage::fvector4 out = {};
+					if (g_pointers->m_gta.m_get_ped_bone(plyr->get_ped(), out, bone_ids[i]))
+					{
+						bones.push_back(out);
+					}
+				}
+
+				if (bones.size() == skel_size)
+				{
+					rage::fvector2 bones_screen[skel_size] = {};
+					ImVec2 render_pos[skel_size]           = {};
+					bool render_skeleton                   = true;
+
+					for (int i = 0; i < skel_size; i++)
+					{
+						if (!g_pointers->m_gta.m_get_screen_coords_for_world_coords(bones[i].data, &bones_screen[i].x, &bones_screen[i].y))
+						{
+							render_skeleton = false;
+							break;
+						}
+						else
+						{
+							auto buffer   = ImVec2((float)*g_pointers->m_gta.m_resolution_x * bones_screen[i].x, (float)*g_pointers->m_gta.m_resolution_y * bones_screen[i].y);
```

Comment:
ImVec2 is explicitly float, why are you C-style casting? Shouldn't the compiler do this implicitly?

Reply:
i just copied the resolution stuff from another esp category in the func, as i said put this together in about 10 minutes so that tiny little detail didn't really cross my mind